.travis.yml

```diff
@@ -3,7 +3,7 @@ services:
   - docker
 
 before_install:
-  - docker build -t tanchowelcomeweek/frontend-test -f ./frontend/Dockerfile.dev ./frontend
+  - docker build -t tanchowelcomeweek/frontend-test -f ./src/frontend/Dockerfile.dev ./test/frontend
```

Comment:
The development Dockerfile and the context provided are in different folders, and the test directory doesn't contain a package.json file to copy over and install from, resulting in an error. Would be easier to have the test directory within the frontend folder like the previous time.